manifests/server/statefulSet.yaml

```diff
@@ -99,6 +99,9 @@ spec:
             path: /healthz/ready
             port: 8443
             scheme: HTTPS
+        resources:
+          limits: {}
+          requests: {}
```

Comment:
see above

Reply:
I've tested that empty map is valid and the end result is that no limits are set. This is the same as before.